website/src/components/SideMenu.tsx

```diff
@@ -1,75 +1,84 @@
-import { Button, Card, Text, Tooltip } from "@chakra-ui/react";
+import { Box, Button, Card, Text, Tooltip } from "@chakra-ui/react";
 import { LucideIcon } from "lucide-react";
 import Link from "next/link";
 import { useRouter } from "next/router";
 import { useTranslation } from "next-i18next";
 import { getTypeSafei18nKey } from "src/lib/i18n";
 
-export interface MenuButtonOption {
+import { HEADER_HEIGHT } from "./Header/Header";
+
+export interface SideMenuItem {
   labelID: string;
   pathname: string;
   icon: LucideIcon;
   target?: HTMLAnchorElement["target"];
 }
 
 export interface SideMenuProps {
-  buttonOptions: MenuButtonOption[];
+  items: SideMenuItem[];
 }
 
-export function SideMenu(props: SideMenuProps) {
+export const SIDE_MENU_WIDTH = {
+  MD: `100px`,
+  LG: `280px`,
+};
+
+export function SideMenu({ items }: SideMenuProps) {
   const router = useRouter();
   const { t } = useTranslation();
 
   return (
-    <main className="sticky top-0 sm:h-full">
-      <Card
-        display={{ base: "grid", sm: "flex" }}
-        width={["100%", "100%", "100px", "280px"]}
-        className="grid-cols-3 gap-2 sm:flex-col sm:justify-between p-4 h-full"
+    <Card
+      position={{ base: "relative", md: "fixed" }}
+      p="4"
+      width={{ base: "100%", md: SIDE_MENU_WIDTH.MD, lg: SIDE_MENU_WIDTH.LG }}
+      height={{ base: "auto", md: `calc(100vh - ${HEADER_HEIGHT} - ${1.5 * 2}rem)` }}
+    >
+      <Box
+        as="nav"
+        gap="2"
+        display={{ base: "grid", md: "flex" }}
+        flexDirection="column"
+        className="grid-cols-3 col-span-3"
       >
-        <nav className="grid grid-cols-3 col-span-3 sm:flex sm:flex-col gap-2">
-          {props.buttonOptions.map((item) => {
-            const label = t(getTypeSafei18nKey(item.labelID));
-            return (
-              <Tooltip
+        {items.map((item) => {
+          const label = t(getTypeSafei18nKey(item.labelID));
+          return (
+            <Tooltip
+              key={item.labelID}
+              fontFamily="inter"
+              label={label}
+              placement="right"
+              className="hidden lg:hidden sm:block"
+            >
+              <Link
                 key={item.labelID}
-                fontFamily="inter"
-                label={label}
-                placement="right"
-                className="hidden lg:hidden sm:block"
+                href={item.pathname}
+                target={!item.target ? "_self" : item.target}
```

Comment:
`item.target ?? "_self"`

Reply:
> `item.target ?? "_self"`

That's is the old code, I didn't want to touch it.